tests/commands/test_edit.py

```diff
@@ -1,16 +1,22 @@
 import os
-import sys
+
 import pytest
 
 import papis.database
 
 from papis.testing import TemporaryLibrary, PapisRunner
 
-script = os.path.join(os.path.dirname(__file__), "scripts.py")
+
+def get_mock_script(commandlet: str) -> str:
```

Comment:
```suggestion
def get_mock_script(name: str) -> str:
```
? I'm not sure I've seed `commandlet` used anywhere. Is that a thing? :))